### PR TITLE
[Backport][AUT-1505] Fix lang attribute is replaced by class attribute #354

### PR DESCRIPTION
**Pull request by andreluizmachado**
This implementation is intended to solve the issue: https://oat-sa.atlassian.net/browse/AUT-1505

Please check the steps to reproduce to see the issue.

### CODE REVIEW

**Review by bartlomiejmarszal — COMMENTED** [review]
This is backport? Can we change. title for backport smoothing for better visibility

File: model/sharedStimulus/parser/JsonQtiAttributeParser.php [hunk 1]
```diff
@@ -76,11 +77,22 @@ private function hydrateXInclude(XInclude $xinclude, DOMDocument $document): XIn
...
-            $xinclude->setAttribute(
-                'xml:lang',
-                $document->getElementsByTagName('div')->item(0)->attributes['lang']->nodeValue
+        $languageAttribute = $document->firstChild->getAttribute('xml:lang');
+
+        if (empty($languageAttribute)) {
+            $this->getLogger()->notice(
+                'lang attribute is empty. Impossible to set the Language Attribute',
+                [
+                    'document' => $document->saveXML()
```
Comment (gabrielfs7) [inline model/sharedStimulus/parser/JsonQtiAttributeParser.php:86]: Are we saving the entire XML in the logs? Or is it the path?

**Review by gabrielfs7 — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: model/sharedStimulus/parser/JsonQtiAttributeParser.php (hunk 1 above)
Comment (andreluizmachado) [reply model/sharedStimulus/parser/JsonQtiAttributeParser.php]: https://github.com/oat-sa/extension-tao-mediamanager/pull/356#discussion_r754919647

**Pull request merged**